images/installer/Dockerfile.upi.ci.rhel8

```diff
@@ -0,0 +1,68 @@
+# NOTE: Forked temporarily from Dockerfile.upi.ci to port to rhel8
+# This Dockerfile is used by CI to test UPI platforms for OpenShift Installer
+# It builds an image containing binaries like jq, terraform, awscli, oc, etc. to allow bringing up UPI infrastructure.
+# It also contains the `upi` directory that contains various terraform and cloud formation templates that are used to create infrastructure resources.
+
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN hack/build.sh
+
+FROM registry.ci.openshift.org/ocp/4.8:cli as cli
+
+FROM registry.ci.openshift.org/ocp/4.8:base
+COPY --from=cli /usr/bin/oc /bin/oc
+COPY --from=builder /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
+COPY --from=builder /go/src/github.com/openshift/installer/upi /var/lib/openshift-install/upi
+COPY --from=builder /go/src/github.com/openshift/installer/data/data/rhcos.json /var/lib/openshift-install/rhcos.json
+
+RUN rpm --import https://packages.microsoft.com/keys/microsoft.asc && \
+    sh -c 'echo -e "[azure-cli]\nname=Azure CLI\nbaseurl=https://packages.microsoft.com/yumrepos/azure-cli\nenabled=1\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" >/etc/yum.repos.d/azure-cli.repo' && \
+    sh -c 'echo -e "[google-cloud-sdk]\nname=Google Cloud SDK\nbaseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el7-x86_64\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg\n       https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg" > /etc/yum.repos.d/google-cloud-sdk.repo'
+
+RUN yum update -y && \
+    yum install --setopt=tsflags=nodocs -y \
+      azure-cli \
+      gettext \
+      google-cloud-sdk \
+      gzip \
+      jq \
+      unzip \
+      openssh-clients \
+      openssl \
+      python3-pyOpenSSL \
+      python2-pyyaml \
+      python3-pyyaml \
```

Comment:
Note we can easily get both python2+3 pyyaml now too; there's a *lot* of stuff in openshift/release still using `python` (aka `python2`).